clientgui/ProjectInfoPage.cpp

```diff
@@ -42,10 +42,14 @@
 #include "res/windowsicon.xpm"
 #include "res/macosicon.xpm"
 #include "res/linuxicon.xpm"
+#include "res/androidicon.xpm"
 #include "res/freebsdicon.xpm"
+#include "res/linuxarmicon2.xpm"
 #include "res/atiicon.xpm"
+#include "res/amdicon2.xpm"
 #include "res/nvidiaicon.xpm"
```

Comment:
If you are not going to use the old icon and want to have a new file with new name then remove the old one.

Reply:
I wanted my PR to be non-destructive, at least until previous developers (Charlie and Rom) have had a chance to review.